lib/android/examples/src/ui_test.nit

```diff
@@ -20,44 +20,37 @@ module ui_test is
 	app_version(0, 1, git_revision)
 	app_namespace "org.nitlanguage.ui_test"
 	android_manifest_activity """android:theme="@android:style/Theme.Light""""
```

Comment:
Spaces would be alright yeah, as long as the `"""` are well-separated the grammar should be fine :)

And the line was just above the new target, besides the highlighting was all crapped-up so I could see this one easily, sorry 'bout that ;)